### DISCUSSION TITLE
Hangman chat minigame

### DISCUSSION THREAD

**Discussion by Esoterism** [ORelio/Minecraft-Console-Client#2056] [Q&A] [answered]
How do you set that up properly?

**Comment by ORelio** [comment, marked as answer]
https://github.com/MCCTeam/Minecraft-Console-Client/issues/1283#issuecomment-712396645
> 1. Set your username in `botowners` inside `MinecraftClient.ini`
> 2. Configure the ChatBot in the `[Hangman]` section, in particular, `enabled=true` and `wordsfile`: [hangman-en.txt](https://github.com/ORelio/Minecraft-Console-Client/blob/master/MinecraftClient/config/hangman-en.txt)
> 3. From a bot owner, type `start` in chat to start the game, `stop` in chat to stop playing before the game is over
> 4. If the game does not work, test if that chat messages are correctly detected using [sample-script-with-chatbot.cs](https://github.com/ORelio/Minecraft-Console-Client/blob/master/MinecraftClient/config/sample-script-with-chatbot.cs)
>    - See also: [Internal commands](https://github.com/ORelio/Minecraft-Console-Client/tree/master/MinecraftClient/config#internal-commands): `/script file.cs`
>    - See also: [Detecting chat messages](https://github.com/ORelio/Minecraft-Console-Client/tree/master/MinecraftClient/config#detecting-chat-messages)

**Reply by Esoterism** [reply]
cheers <3
